projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -85,7 +85,6 @@ public function reregister_menu_items() {
 			$this->add_new_site_link();
 		}
 
-		$this->add_woocommerce_installation_menu();
```

Comment:
I think this would work if we decide to put this menu item behind the "Classic style" option. But, the "Admin interface style" setting doesn't currently exist on simple sites, and I think the `option` may even be removed when an Atomic site is reverted. We could still check for the option on simple sites in preparation for it, though. 🤔 
```
if ( get_option( 'wpcom_admin_interface' ) !== 'wp-admin' ) {
	$this->add_woocommerce_installation_menu();
}
```